src/main/kotlin/no/nav/syfo/services/ArenaDialogmeldingService.kt

```diff
@@ -44,6 +46,10 @@ class ArenaDialogmeldingService(
         )
     }
 
+    suspend fun isMeldingStoredInModia(msgId: String): Boolean {
```

Comment:
Lurte på om vi burde endre til `isMeldingInBehandlerdialog` f.eks? For ikke å forvirre med evt dialogmøte-dialogmeldinger som vi ikke sjekker etter her :thought_balloon:

Reply:
Ja, vi kan være litt mer presise 👍🏼